client/app/scripts/superdesk-desks/views/aggregate-settings.html

```diff
@@ -40,20 +40,39 @@ <h3 translate>Monitoring settings</h3>
 
     <div sd-wizard-step data-title="{{ :: 'Saved Searches' | translate }}" data-code="searches">
         <div class="content">
-            <div class="legend" translate>Select saved searches for view.  
-                <span sd-switch class="pull-right" ng-model="showAllSavedSearches" 
-                 ng-click="initSavedSearches(showAllSavedSearches)" title="{{'Show all saved searches'}} | translate"></span>
+            <div class="legend" translate>Select global saved searches  
+                <span sd-switch class="pull-right" ng-model="showGlobalSavedSearches" 
+                 title="{{'Show all saved searches'}} | translate"></span>
             </div>
-            <div class="desk" ng-repeat="search in currentSavedSearches | orderBy: 'name'">
+            <div class="desk" ng-repeat="search in globalSavedSearches | orderBy: 'name'" ng-if="showGlobalSavedSearches">
```

Comment:
Can the `globalSavedSearches` list change? If so, we might want to add `track by search._id` here.